SampleCode/LabelingUX/Client/src/utils/documentLoader/pdfLoader.ts

```diff
@@ -18,6 +18,7 @@ export class PdfLoader implements IDocumentLoader {
 
     public async setup(): Promise<void> {
         this.pdf = await pdfjsLib.getDocument({
+            isEvalSupported: false,
```

Comment:
This is used to resolve the the high vulnerability with pdfjs-dist as highlighted after running npm audit.
More details are found here : https://github.com/advisories/GHSA-wgrm-67xf-hhpq